.buildkite/sanitizers.yml

```diff
@@ -0,0 +1,31 @@
+# These steps should only run on `sandbox.jl` machines, not `docker`-isolated ones
+# since we need nestable sandboxing.  The rootfs images being used here are built from
+# the `.buildkite/rootfs_images/llvm-passes.jl` file.
+agents:
+  queue: "julia"
+  # Only run on `sandbox.jl` machines (not `docker`-isolated ones) since we need nestable sandboxing
+  sandbox.jl: "true"
+  os: "linux"
+
+steps:
+  - label: "asan"
+    key: asan
+    plugins:
+      - JuliaCI/julia#v1:
+          version: 1.6
```

Comment:
Is `julia` command unavailable in `$PATH`? I get:

> contrib/check-asan.jl: line 12: exec: julia: not found

https://buildkite.com/julialang/julia-master/builds/1815#2ea863e5-e927-484d-a64d-0c0bf51346d7

Reply:
There are multiple layers of sand boxing happening here. We need to add “/cache” as a mapping to the workspaces to the sandbox plugin. I’m AFK right now otherwise I would do it.

Reply:
Maybe I'm misunderstanding, but which Julia binary are you trying to run here? Presumably you're trying to run the Julia binary that you just built from source, right? So shouldn't you do e.g. `./tmp/julia`?

Reply:
Ah, sorry it was unclear. I want to use `julia` from `JuliaCI/julia#v1` to run the script `check-asan.jl` which then in turn runs `./tmp/test-asan/usr/bin/julia-debug`. I want some stable `julia` binary just for running the script. (I could do it in bash but it's rather too complicated)